### ISSUE TITLE
Page resize ruins order of list

### ISSUE THREAD

**Issue by techvet** [litera/jquery-transpose#2]
I noticed once a page is re-sized that the order of a transposed item reverts to being out of order again. I attempted to wire up the transpose() function to fire when the page is re-sized but I realized that the transpose method can only reorder an item once. I will fork the repo and attempt to find a workaround, so far the only thing that has worked is wrapping the item in a fixed width div but this ruins the responsiveness of the site.

**Comment by litera** [comment]
Yes at current state of affairs plugin only reorders elements a single time. This means that resizing the page re-floats them breaking column ordering. I will likely upgrade plugin to allow reordering.
